src/main/java/core/basesyntax/lib/Injector.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.lib;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.exceptions.NoAnnotationException;
+import core.basesyntax.factory.Factory;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+
+public class Injector {
+
+    public static Object getInstance(Class clazz) throws NoSuchMethodException,
+            IllegalAccessException, InvocationTargetException,
+            InstantiationException, RuntimeException {
```

Comment:
```suggestion
    public static Object getInstance(Class clazz) throws NoSuchMethodException,
            IllegalAccessException, InvocationTargetException,
            InstantiationException {
```
we should not indicate RuntimeExceptions in method signature